src/server/utils/getIntegratron.js

```diff
@@ -0,0 +1,9 @@
+import Queue from 'server/utils/bull';
+
+let integratron;
```

Comment:
Ah, this is how you decided to handle it. It was either going to be like this or create an object cache in `bull.js`. Seems reasonable for starters.